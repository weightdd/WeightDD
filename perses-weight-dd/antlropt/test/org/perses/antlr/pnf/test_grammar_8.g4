grammar test_grammar_8;

start
    : rule*
    | ID*
    | 'test'*
    | ('A' | 'B')*
    | ('C' | 'D' | 'E')*
    | ('F' | 'G' | another)*
    ;