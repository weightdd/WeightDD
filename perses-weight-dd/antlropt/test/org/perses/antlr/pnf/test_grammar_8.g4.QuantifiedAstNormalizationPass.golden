// Auto-generated by Perses AST.
// DO NOT MODIFY.
grammar test_grammar_8;



start
    : aux_rule__start_10?
    ;

kleene_plus__start_1
    : rule+
    ;

kleene_plus__start_2
    : ID+
    ;

kleene_plus__start_3
    : 'test'+
    ;

aux_rule__start_4
    : 'A'
    | 'B'
    ;

kleene_plus__start_5
    : aux_rule__start_4+
    ;

aux_rule__start_6
    : 'C'
    | 'D'
    | 'E'
    ;

kleene_plus__start_7
    : aux_rule__start_6+
    ;

aux_rule__start_8
    : 'F'
    | 'G'
    | another
    ;

kleene_plus__start_9
    : aux_rule__start_8+
    ;

aux_rule__start_10
    : kleene_plus__start_1
    | kleene_plus__start_2
    | kleene_plus__start_3
    | kleene_plus__start_5
    | kleene_plus__start_7
    | kleene_plus__start_9
    ;

